tutorials/tree/dictionary/readTree.cpp

```diff
@@ -0,0 +1,51 @@
+// Author: Alvaro Tolosa-Delgado CERN 07/2023
+// Author: Jorge Agramunt Ros IFIC(Valencia,Spain) 07/2023
+
+/*************************************************************************
+ * Copyright (C) 1995-2023, Rene Brun and Fons Rademakers.               *
+ * All rights reserved.                                                  *
+ *                                                                       *
+ * For the licensing terms see $ROOTSYS/LICENSE.                         *
+ * For the list of contributors see $ROOTSYS/README/CREDITS.             *
+ *************************************************************************/
+
+#include <iostream>
+
+#include <TFile.h>
+#include <TTree.h>
+#include <TTreeReader.h>
+#include <TTreeReaderValue.h>
+
+#include "data2Tree.hpp"
+
+void readTree()
+{
+
+   std::unique_ptr<TFile> ifile = std::make_unique<TFile>("testFile.root", "read");
+   if ( nullptr == ifile ) {
+      std::cerr << " File not found " << std::endl;
+      return;
+   }
+   if (!TClass::GetClass(typeid(myDetectorData))->IsLoaded()) {
+      std::cerr << " TClass::GetClass(typeid(myDetectorData))->IsLoaded() == false " << std::endl;
+   }
```

Comment:
Is this something we expect the user to want to use in all scripts or is it just for debugging purposes?

Reply:
I do not remember why I put that, it was long time ago. I can remove it.